themes/default/content/templates/container-service/_index.md

```diff
@@ -1,8 +1,9 @@
 ---
+title_tag: Templates for Deploying Container Services
 title: Container Service Templates
 layout: overview
 description: Pulumi program templates are the fastest way to deploy container services on AWS, Azure, or Google Cloud Platform. Templates come with predefined infrastructure as code so you can get started instantly.
-meta_desc: Pulumi program templates that make it easy to deploy container services on AWS, Azure, or Google Cloud Platform.
+meta_desc: Instantly deploy container services on AWS, Azure, or Google Cloud Platform with Pulumi Container Service templates. 
```

Comment:
```suggestion
meta_desc: Instantly deploy container services on AWS, Azure, or Google Cloud Platform with Pulumi Container Service templates.
```

Reply:
```suggestion
meta_desc: Easily deploy container services on AWS, Azure, or Google Cloud Platform with Pulumi Container Service templates. 
```

Reply:
"Easily" is being used vs "Instantly" in most all of the other template description changes being proposed. I also feel it is a far better word to use here and more consistent with the original intent of the statement IMO.